paasta_tools/paasta_cli/cmds/rollback.py

```diff
@@ -37,9 +38,9 @@ def add_subparser(subparsers):
                              required=True,
                              )
     list_parser.add_argument('-i', '--instance',
-                             help='Mark the instance we want to roll back (e.g. '
-                             'canary, .main)',
-                             required=True,
+                             help='Mark the instance[s] we want to roll back (e.g. '
+                             'canary, main)',
```

Comment:
Can you remove the space to make it more obvious it is a csv? Also can you add more words to describe that you can describe a single instance or mulitple instance in csv form. Also can you describe the default behavior? Helpstrings are cheap and good.